var/spack/repos/builtin/packages/msprime/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Msprime(PythonPackage):
+    """The library is a reimplementation of Hudson’s seminal ms program,
```

Comment:
```suggestion
    """The library is a reimplementation of Hudson's seminal ms program,
```
This should fix failing Python 2 tests